app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -119,6 +128,32 @@ public function store(Request $request, $id)
             $return_msg = trans('admin/users/message.declined');
         }
 
+        $item = $acceptance->checkoutable_type::find($acceptance->checkoutable_id);
+        $assigned_to = User::find($item->assigned_to);
```

Comment:
Will probably  need a switch statement for accessory acceptance. some table headers are different and some columns should also be included. But this covers assets